Content.Client/Kitchen/UI/GrinderMenu.xaml

```diff
@@ -3,7 +3,11 @@
             xmlns:controls="clr-namespace:Content.Client.UserInterface.Controls"
             Title="{Loc grinder-menu-title}" MinSize="768 256">
     <BoxContainer Orientation="Horizontal">
-        <BoxContainer Orientation="Vertical" VerticalAlignment="Top" Margin="8">
+        <BoxContainer Orientation="Vertical" VerticalAlignment="Top" Margin="8" MinWidth="100">
+            <Label Text="{Loc grinder-menu-auto-label}" HorizontalAlignment="Center"/>
+            <Button Name="AutoModeButton" Text="{Loc grinder-menu-auto-button}" TextAlign="Center" MinSize="64 48"/>
+            <Control MinSize="0 16"/>
+            <Label Text="{Loc grinder-menu-manual-label}" HorizontalAlignment="Center"/>
```

Comment:
Use a margin instead of a dummy control.

Reply:
Fixed